iree/hal/cuda/CMakeLists.txt

```
@@ -82,6 +83,22 @@ iree_cc_library(
   PUBLIC
 )
 
+if(IREE_HAL_DRIVER_CUDA AND IREE_ENABLE_RUNTIME_TRACING)
```

Please fix bazel->cmake by adding this on the bazel side or using `iree_cmake_extra_content`. I would do the former for now and we'll find out if it works when importing into the google monorepo and disable it then if necessary, I guess. Also, this whole file returns early if cuda isn't enabled, so the extra condition here isn't necessary.

```suggestion
if(IREE_ENABLE_RUNTIME_TRACING)
```

---

There is no support for CUPTI with the bazel build due to the requirement on system libraries. It sounds like `iree_cmake_extra_content` is actually what I need here.